doc/HOWTO_RELEASE.rst.txt

```diff
@@ -418,7 +418,7 @@ wheels with your GPG key before upload.  ``-r warehouse`` causes the upload to
 use the Warehouse PyPI server.  This is a good idea because the Warehouse
 server seems to be a lot more reliable in receiving automated wheel uploads.
 For this flag to work, you will need a ``warehouse`` section in your
-``~/.pypirc`` file, of form:
+``~/.pypirc`` file, of form::
```

Comment:
ISTR that the `.pypirc` file is no longer needed since the PyPI upgrade. Maybe this whole section is outdated. @njsmith @rgommers Comment?

Reply:
Indeed. `.pypirc` not needed and the whole thing about Warehouse can be removed. Warehouse was what is now the default PyPI implementation.